src/lua_generator.cpp

```diff
@@ -0,0 +1,769 @@
+/*
```

Comment:
maybe have a common naming scheme for these files, much like old generators all start with `idl_gen_` ? `bfbs_gen_` ?

Reply:
Done.